lib/utils/ruleRawsAfterIndex.mjs

```diff
@@ -0,0 +1,13 @@
+/**
+ * @param {import('postcss').Rule} node
+ * @returns {number}
+ */
+export default function ruleRawsAfterIndex(node) {
+	// always subtract 1 for `}`
```

Comment:
This is similar to `@media screen` where we have to do `+1` to account for the `@` when calculating any index after the name.